de/manual/ch_Business.docbook

```diff
@@ -1,1279 +1,1976 @@
+            </para>
+          </listitem>
+        </itemizedlist>
+      </para>
+    </abstract>
 
-    <para>Customers are people or companies to whom you sell goods or services. They must be registered within
-      the A/R system.
+    <para>In den folgenden Abschnitten wird die Arbeit mit den einzelnen Komponenten ausführlicher
+      vorgestellt.
     </para>
 
     <sect2 id="busnss-ar-custnew2">
-      <title>New</title>
-
-      <para>To register a new customer, enter the menu &mc.bus.cus.new-cus; . Fill in customer information, such
-        as Company Name, Address, Phone, Fax, etc. Below is a list of the other options:
-      </para>
-
-      <itemizedlist>
-        <listitem>
-          <para><guilabel>Identification - Customer Number</guilabel> - can be any number by which you would like to
-            refer to this customer. You may leave it blank and a number will be chosen
-            automatically. See <xref linkend="counters-book-options" /> for more info.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Identification - Active</guilabel> - differentiates active customers from inactive ones.
-            This is useful when you have many past customers, and you want to see only those marked
-            active.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Billing Address - Name</guilabel> - is the contact name of the person to receive the
-            invoices.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Notes</guilabel> - records any additional comments about the customer.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Billing Information - Currency</guilabel> - specifies the billing currency for this
-            customer. It will default to the default currency set in
-            <guilabel>Preferences</guilabel>
-          </para>
+      <title>Neuer Kunde</title>
+
+      <para>Um einen Datensatz für einen neuen Kunden anzulegen, öffnen Sie den Dialog &mc.bus.cus.new-cus;.
+        Erfassen Sie die für Sie relevanten Informationen, wie z.B. Name des Kundens, Adresse,
+        Telefonnummer, E-Mail etc. In dem Dialog können Sie auf drei Reitern die notwendigen Daten
+        eingeben.
+      </para>
+
+      <table pgwide="1" id="busnss-new-customer">
+        <title>Dialog <guilabel>Neuer Kunde</guilabel></title>
+
+        <tgroup align="left" cols="3">
+          <colspec colwidth="0.33*"/>
+
+          <colspec colwidth="0.33*"/>
+
+          <colspec colwidth="0.33*"/>
+
+          <tbody>
+            <row>
+              <entry>
+                <screenshot>
+                  <screeninfo>432x687</screeninfo>
+
+                  <mediaobject>
+                    <imageobject>
+                      <imagedata fileref="figures/Busnss_Customer_New_1.png"
+                                 srccredit="Christian Wehling" width="100%" />
```

Comment:
In den neuen Bildern wurden wieder Ränder eingefügt. Wa das Absicht?

Reply:
Die Ränder werden durch das `figure`- Tag eingefügt. Die sind jedoch nur in Yelp sichtbar.

Reply:
Die Tage war unten in der Vorschau noch eine karierte Fläche, also Transparenter Hintergrund um das eigentliche Bild. Der wird dann gern von einigen Programmen als schicker Schatten dargestellt.